earth_enterprise/src/common/tests/SConscript

```diff
@@ -50,8 +51,12 @@ map(lambda test: env.test(test, [test + '.cc'],
     ['etencoder_tests',
      ])
 
+testdir = os.path.join(env.exportdirs['bin'],'tests')
+env.copyfile(testdir, 'RunAllTests.pl')
 
-env.copyfile(os.path.join(env.exportdirs['bin'],'tests'),
-             'RunAllTests.pl')
+# Drop in the test data
+fusiontestdir = os.path.join(testdir, 'fusion')
+shutil.rmtree(fusiontestdir, ignore_errors=True)
+shutil.copytree('../../fusion', fusiontestdir)
```

Comment:
Thank you. 
I believe you need to copy only src/fusion/testdata.

Also, please see comment at #15. We may skip copying of testdata if we make a message more informative, e.g. suggesting to run tests from src directory.
e.g.  for this tests:
Running idl_tests ...                      FAILED
----------
[==========] Running 10 tests from 1 test case.
[----------] Global test environment set-up.
common/khxml/idl_tests.cc:39: Failure
we can just add message
https://github.com/google/earthenterprise/blob/master/earth_enterprise/src/common/khxml/idl_tests.cc#L39

ASSERT_TRUE(khDirExists(source_dir_)) << "Couldn't find testdata dir: " << source_dir_ << ". Try to run test from src-directory";

And Assert like that can be added to other tests.